packages/static-wado-creator/lib/operation/ThumbnailService.js

```diff
@@ -99,6 +105,22 @@ class ThumbnailService {
   generateThumbnails(dataSet, metadata, callback) {
     const { imageFrame, id } = this.favoriteThumbnailObj;
 
+    // There are various reasons no thumbnails might be generated, so just return
+    if( !id ) {
+      const pixelData = metadata[Tags.PixelData];
+      if( pixelData ) {
+        const { BulkDataURI } = pixelData;
+        if( BulkDataURI?.indexOf("mp4") ) {
+          console.log("MP4 - converting video format");
+          
+        } else {
+          console.log('pixelData = ', pixelData, Tags.PixelData);
```

Comment:
Just add some logging to figure out what is going on - normally I only hit the MP4 or the series is of other type message.